### ISSUE TITLE
[java] AtLeastOneConstructorRule: false-positive with lombok.AllArgsConstructor

### ISSUE THREAD

**Issue by dylanleung** [pmd/pmd#1547]
java
**Affects PMD Version:** 
6.7
**Rule:**
AtLeastOneConstructorRule
**Description:**
what means AllArgsConstructorAtLeastOneConstructor?

**Code Sample demonstrating the issue:**

```
@Override
    protected Collection<String> defaultSuppressionAnnotations() {
        return Arrays.asList("lombok.Data",
                "lombok.Value",
                "lombok.Builder",
                "lombok.NoArgsConstructor",
                "lombok.RequiredArgsConstructor",
                "lombok.AllArgsConstructorAtLeastOneConstructor");
    }

```

**Running PMD through:** *[CLI | Ant | Maven | Gradle | Designer | Other]*
IDEA

**Comment by adangel** [comment]
I guess that's a copy-paste-error. There is no lombok annotation with that name documented: https://projectlombok.org/api/lombok/package-summary.html

For whoever want's to contribute:
1. Create a test case using the lombok annotation "lombok.AllArgsConstructor"
2. See, that this creates a false-positive
3. Fix the rule, the test case should now run.
